modules/ipi-install-replacing-a-bare-metal-control-plane-node.adoc

```diff
@@ -15,8 +15,33 @@ If you reuse the `BareMetalHost` object definition from an existing control plan
 Existing control plane `BareMetalHost` objects may have the `externallyProvisioned` flag set to `true` if they were provisioned by the {product-title} installation program.
 ====
 
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
```

Comment:
Can you put `cluster-admin` in backticks? (other places in our docs that use this wording do)

Reply:
This also aligns with the the [markup reference](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference).